tests/wpunit/Tribe/AdminHelpersTest.php

```diff
@@ -0,0 +1,262 @@
+<?php
+
+namespace Tribe;
+
+require_once codecept_data_dir( 'classes/WP_Screen.php' );
+
+use Tribe\Common\Tests\WP_Screen as WP_Screen;
+use Tribe__Context as Context;
+use Tribe__Admin__Helpers as Admin_Helpers;
+
+class AdminHelpersTest extends \Codeception\TestCase\WPTestCase {
+
+	function setUp() {
+		parent::setUp();
+
+		global $pagenow;
+		$pagenow = null;
+		unset( $_GET['post_type'] );
+
+	}
+
+	/**
+	 * @test
+	 * it should be instantiatable
+	 * @since TBD
+	 */
+	public function it_should_be_instantiatable() {
+		$helpers = $this->make_instance();
+
+		$this->assertInstanceOf( Admin_helpers::class, $helpers );
+	}
+
+	/**
+	 * Return instance
+	 *
+	 * @return Admin_Helpers
+	 * @since TBD
+	 */
+	private function make_instance() {
+		return new Admin_Helpers();
+	}
+
+	/**
+	 * Check results for `is_post_type_screen` on the new event page
+	 *
+	 * @since TBD
+	 * @test
+	 */
+	public function is_post_type_tribe_events() {
+		$helpers = $this->make_instance();
+		$helpers->doing_test( true );
```

Comment:
Following along the filter here I would use `add_filter( 'tribe_admin_is_wp_screen', '__return_true' );`